app/javascript/lib/metrics-api.js

```diff
@@ -200,6 +201,12 @@ export function log(name, props={}) {
     props['featuredSpace'] = window.SCP.featuredSpace
   }
 
+  if ('SCP' in window && window.location.search.includes('scpbr=')) {
```

Comment:
Why do you need the check for window.location.search.includes('scpbr=')?  (or the check for SCP in window?)

 Could you do 
```
const brandingGroup = getBrandingGroup() 
props['brand'] = brandingGroup ? brandingGroup : ''
```
instead?

Reply:
Good point, done in a08cd64.